iocage/cli/chroot.py

```diff
@@ -64,32 +65,36 @@ def chroot_cmd(jail, command):
             lgr.error("  {} ({})".format(u, t))
         raise RuntimeError()
     else:
-        raise RuntimeError("{} not found!".format(jail))
+        lgr.error("{} not found!".format(jail))
+        exit(1)
 
     devfs_stderr = mount(f"{path}/root/dev", "devfs")
 
     if devfs_stderr:
-        raise RuntimeError("ERROR: Mounting devfs failed!")
+        lgr.critical("Mounting devfs failed!")
+        exit(1)
 
     fstab_stderr = mount(f"{path}/fstab", "fstab")
 
     if fstab_stderr:
-        raise RuntimeError("ERROR: Mounting devfs failed!")
+        lgr.critical("Mounting devfs failed!")
+        exit(1)
 
     chroot = Popen(["chroot", f"{path}/root"] + command)
     chroot.communicate()
 
     udevfs_stderr = umount(f"{path}/root/dev", "devfs")
     if udevfs_stderr:
-        raise RuntimeError("ERROR: Unmounting devfs failed!")
+        lgr.critical("Unmounting devfs failed!")
+        exit(1)
 
     ufstab_stderr = umount(f"{path}/fstab", "fstab")
     if ufstab_stderr:
         if b"fstab reading failure\n" in ufstab_stderr:
             # By default our fstab is empty and will throw this error.
             pass
         else:
-            raise RuntimeError("ERROR: Unmounting fstab failed!")
+            lgr.critical("Unmounting fstab failed!")
 
     if chroot.returncode:
-        lgr.warning("WARNING: Chroot had a non-zero exit code!")
+        lgr.error("Chroot had a non-zero exit code!")
```

Comment:
Should still be a warning, we're not sure if that's intentional or not with whatever the user is doing.